tests/lib/sleep.ts

```diff
@@ -0,0 +1,3 @@
+export default async function sleep(ms: number): Promise<void> {
+	return new Promise((resolve) => setTimeout(resolve, ms));
+}
```

Comment:
```suggestion
export const sleep = util.promisify(setTimeout);
}
```
btw `import { setTimeout } from 'timers/promises'` would be better but it's not available in the nodejs 12 used here (but updating nodejs may well make sense)